src/test/java/teammates/test/pageobjects/AdminEmailPage.java

```diff
@@ -17,6 +17,7 @@ protected boolean containsExpectedPageContents() {
 
     public void inputRecipient(String recipient) {
         WebElement recipientBox = this.getRecipientBox();
+        recipientBox.clear();
```

Comment:
` emailPage.inputRecipient("recipient@email.tmt");` 
because the statement appends to the previous input in the field (`emailPage.inputRecipient("recipient");`) and the input becomes 'recipientrecipient@email.tmt'